### PR TITLE
Move to vsc-install 0.9.1, remove more unnecessary code

### PR DESCRIPTION
**Pull request by stdweird**

### CODE REVIEW

**Comment by boegel** [comment]
Looks fine.

**Comment by boegel** [comment]
Jenkins: test this please

File: setup.py [hunk 1]
```diff
@@ -36,17 +36,13 @@
 import vsc.install.shared_setup as shared_setup
 from vsc.install.shared_setup import ag, kh, jt, sdw, URL_GH_HPCUGENT
 
-# Re-reload the vsc modules that vsc-base ships
-shared_setup.RELOAD_VSC_MODS = True
```
Comment (boegel) [inline setup.py]: @stdweird: why is this no longer needed now?

Comment (stdweird) [inline setup.py]: becasue it works without :smile: i probably made this workaround for something caused by another problem

Comment (boegel) [inline setup.py]: ok

**Comment by boegel** [comment]
Thanks @stdweird!

**Pull request merged**